test/filters/oidc/oidc_filter_test.cc

```diff
@@ -1128,7 +1138,9 @@ google::rpc::Code OidcFilterTest::MakeRequestWhichWillCauseTokenRetrieval(
   auto raw_http = common::http::response_t(
       new beast::http::response<beast::http::string_body>());
   raw_http->result(beast::http::status::ok);
-  EXPECT_CALL(*mocked_http, Post(Eq(token_uri), _, _, Eq("some-ca"),
+  const common::http::TransportSocketOptions opt{.ca_cert_ = "some-ca",
```

Comment:
will it make sense to have a `kDefaultTransportSocketOpts` constant defined in the beginning?